src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/miner/IndustrialMiner.java

```diff
@@ -202,22 +203,23 @@ public void onInteract(Player p, Block b) {
     }
 
     /**
-     * This returns whether this {@link IndustrialMiner} can mine the given {@link Material}.
+     * This returns whether this {@link IndustrialMiner} can mine the given {@link Block}.
+     *
+     * @param block
+     *            The {@link Block} to check
      * 
-     * @param type
-     *            The {@link Material} to check
-     * 
-     * @return Whether this {@link IndustrialMiner} is capable of mining this {@link Material}
+     * @return Whether this {@link IndustrialMiner} is capable of mining this {@link Block}
      */
-    public boolean canMine(@Nonnull Material type) {
+    public boolean canMine(@Nonnull Block block) {
         MinecraftVersion version = Slimefun.getMinecraftVersion();
+        Material type = block.getType();
 
-        if (type == Material.ANCIENT_DEBRIS) {
-            return canMineAncientDebris.getValue();
+        if (version.isAtLeast(MinecraftVersion.MINECRAFT_1_16) && type == Material.ANCIENT_DEBRIS) {
```

Comment:
Ah, hold up, we dont need a version check here for 1.16 anymore

Reply:
alrighty, I'm already on outdated on this pull request for many months now, kinda rusty on mc development